modules/installing-gitops-operator-in-web-console.adoc

```diff
@@ -11,6 +11,7 @@
 * Access to the {product-title} web console.
 * An account with the `cluster-admin` role.
 * You are logged in to the {product-title} cluster as an administrator.
+* Your cluster has the xref:../../post_installation_configuration/cluster-capabilities.adoc#cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
```

Comment:
`xref` links are not allowed in module files in the OCP docs repo. Please remove the xref from this module file. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references